e2e/rdctl.e2e.spec.ts

```diff
@@ -314,11 +317,45 @@ test.describe('Command server', () => {
     expect(resp.ok).toBeTruthy();
     const telemetry = (await resp.json() as Settings).application.telemetry.enabled;
 
-    resp = await doRequest('/v0/settings', JSON.stringify({ application: { telemetry: { enabled: !telemetry } } }), 'PUT');
+    resp = await doRequest('/v0/settings', JSON.stringify({ version: CURRENT_SETTINGS_VERSION, application: { telemetry: { enabled: !telemetry } } }), 'PUT');
     expect(resp.ok).toBeTruthy();
     await expect(resp.text()).resolves.toContain('no restart required');
   });
 
+  test('should complain about a missing version field', async() => {
+    let resp = await doRequest('/v0/settings');
+
+    expect(resp.ok).toBeTruthy();
+
+    const body: RecursivePartial<Settings> = await resp.json() as Settings;
```

Comment:
```suggestion
    const body: RecursivePartial<Settings> = await resp.json();
```
Because `.json()` returns `Promise<any>`, we don't need any explicit casts to convert it to any other type.

Reply:
ok